lib/ciChecks.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Check for CI test failure and pings PR author.
+ * @param {import('probot').Context} context
+ */
+const handleFailure = async (context) => {
+  /**
+   * @type {import('probot').Octokit.ChecksGetSuiteResponse} checkSuite
+   */
+  const checkSuite = context.payload.check_suite;
+  if (checkSuite.conclusion === 'failure') {
+    // A checksuite might fail due to a merge to develop. We don't need to
+    // check for these since they are not failures related to a pull request.
+    if (checkSuite.pull_requests.length > 0) {
```

Comment:
I did not understand this part - are you talking about the failure which happens when we merge to develop before running tests?

Reply:
Ah ok - thanks for explaining. Add your reply to the comment as well.

Reply:
Done.